testing/trees.R

```diff
@@ -0,0 +1,289 @@
+required <- list("rpart", "ranger", "randomForest", "maptree", "cluster", "icebox", "ggplot2", "plotly")
+new.pkgs <- required[!(required %in% installed.packages()[,"Package"])]
+if(length(new.packages) > 0) install.packages(new.pkgs)
+
+install.packages(c("rpart", "ranger", "randomForest", "maptree", "cluster", "icebox", "ggplot2", "plotly"))
+
+# Load packages
+lapply(required, require, character.only = TRUE)
+
+################################
+# CHECKS 
+################################
+
+checks <- function(type, y, train, test = NULL, ...) {
+  
+  train = if(is.data.frame(train)) train else as.data.frame(train) 
+  
+  if(type == "classif") {
+    
+    train[ycol] <- as.factor[train[ycol]]
+    
+  } else if(type == "surv"){
+    
+    train[ycol] <- train[ycol] == y[["success"]]
+    train[y[["time"]]] <- as.numeric(train[y[["time"]]])
+    
+  } else{
+    
+    train[ycol] <- as.numeric(train[ycol])
+    
+  }
+}
+
+################################
+# CART
+################################
+
+wrapperCART <- function(type, y, train, test = NULL, ordered.vars = NULL, count = F,
```

Comment:
Put type after train (so it's not essential) and specify possible options
```suggestion
wrapperCART <- function(y, train, type = c('regr', 'classif', 'surv'), test = NULL, ordered.vars = NULL, count = F,
```